templates/flavors/machinepool-windows/machine-pool-deployment-windows.yaml

```diff
@@ -25,7 +25,7 @@ kind: AzureMachinePool
 metadata:
   name: "${CLUSTER_NAME}-mp-win"
   annotations:
-    runtime: dockershim
+    runtime: containerd
```

Comment:
same here, should be replaced with https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/main/templates/flavors/machinepool-windows-containerd

Reply:
done